packages/core/src/components/pill/pill.tsx

```diff
@@ -95,7 +103,7 @@ export class Pill implements IxComponent {
               }
             : {}
         }
-        title={this.hostElement.textContent}
+        title={determineTooltip(this.tooltipText, this.hostElement.textContent)}
```

Comment:
`title` attribute will not display the styled `ix-tooltip`. At least we should use our own tooltip to provide a consistent ui